DOAX-VenusVacation/d3dx.ini

```diff
@@ -1369,38 +1401,135 @@ run = CommandListClothes
 ; Bytecode hash: 8479a35b
 hash = 5af6c9fd4d23c6e0
 run = CommandListClothes
+[ShaderOverrideClothesPS8-20200610]
+; Bytecode hash: 7a8213e2
+hash = dd9678efa5efd982
+run = CommandListClothes
 
-[ShaderOverrideClothesPS9]
-; Cherry Blossoms Heart
+[ShaderOverrideClothesPS9-Old]
+; Cherry Blossoms Heart, Kasumi Martini Gemstone, Martini SR Hardware
 ; Bytecode hash: b0adc5a2
 hash = e6a0802f6b5a0285
 run = CommandListClothes
-[ShaderOverrideClothesPS10]
-; Common Mermaid | Martini SR SSR
+[ShaderOverrideClothesPS9-20200610]
+; Bytecode hash: 72899d57
+hash = 52c5ae3cdc7810ca
+run = CommandListClothes
+
+[ShaderOverrideClothesPS10-Old]
+; Common Mermaid | Martini SR SSR | Reindeer Bell
 ; Bytecode hash: 75e3e462
 hash = 7633276941ec4f7a
 run = CommandListClothes
-[ShaderOverrideClothesPS11]
+[ShaderOverrideClothesPS10-20200610]
+; Bytecode hash: 3ea17e58
+hash = c46741f26dabb5fd
+run = CommandListClothes
+
+[ShaderOverrideClothesPS11-Old]
 ; Transparent white cloth on black bikini
 ; Bytecode hash: c42e8e56
 hash = 746c0cfeba78dbd9
 run = CommandListClothes
+[ShaderOverrideClothesPS11-20200610]
+; Bytecode hash: f6c01e51
+hash = 1a57b0790b876d57
+run = CommandListClothes
 
 [ShaderOverrideClothesPS12-Old]
-; Destiny Child bikini text (?)
+; Destiny Child bikini text, Dusk Set Frills
 ; Bytecode hash: ab5e9fae
 hash = 088290cf9c3bf97e
 run = CommandListClothes
 [ShaderOverrideClothesPS12-20190425]
 ; Bytecode hash: 21cf1e72
 hash = 334a305914e33465
 run = CommandListClothes
+[ShaderOverrideClothesPS12-20200610]
+; Bytecode hash: df34aecb
+hash = 5c4ca403dd607603
+run = CommandListClothes
 
-[ShaderOverrideClothesPS13]
-; Hitomi Jeans SR
+[ShaderOverrideClothesPS13-Old]
+; Hitomi Jeans SR (top of jeans)
 ; Bytecode hash: 1ead111f
 hash = 074cff4a129d961c
 run = CommandListClothes
+[ShaderOverrideClothesPS13-20200610]
+; Bytecode hash: f92db6e8
+hash = 635329c855070942
+run = CommandListClothes
+
+[ShaderOverrideAccessoryLensePS-Old]
+; Glasses and Demist
+; Bytecodde hash: 3da2a1ac
+hash = 1e372bfddbafcfc9
+run = CommandListClothes
+allow_duplicate_hash=overrule
+[ShaderOverrideAccessoryLensePS-20200610]
+; Bytecode hash: 403a7529
+hash = 90413848ce1de931
+run = CommandListClothes
+allow_duplicate_hash=overrule

```

Comment:
I guess if this shader is new, nothing else should be using this yet, so do we really need allow_duplicate_hash here? Or is this already in use in other mods?

Reply:
This is the same hash that's always been in demist, which other people had already published interim fixes for with the same shader hash.